### ISSUE TITLE
graphql-server-koa not working

### ISSUE THREAD

**Issue by AlexLeung** [apollostack/graphql-server#231]
Given the following code:
```
import koa from 'koa';
import koaRouter from 'koa-router';
import { graphqlKoa, graphiqlKoa } from 'graphql-server-koa';
import koaBody from 'koa-body';
import schema from './schema';

const app = new koa();
const router = new koaRouter();
const PORT = 3010;
app.use(koaBody());
router.post('/graphql', graphqlKoa({schema}));
router.get('/graphiql', graphiqlKoa({ endpointURL: '/graphql', query: '' }));
app.use(router.routes());
app.use(router.allowedMethods());
app.listen(PORT);
````

When trying a `POST` request to /graphql, I get the following error 
``` 
TypeError: Cannot read property 'body' of undefined
      at C:\Users\Alex\...\node_modules\graphql-server-koa\src\koaApollo.ts:38:21
```

When trying a `GET` request to /graphiql, I get the following error
```
TypeError: Cannot read property 'query' of undefined
      at Object.<anonymous> (C:\Users\Alex\...\node_modules\graphql-server-koa\src\koaApollo.ts:119:26)
```
This error occurs with both the `koa` and `koa@2` packages.

**Comment by AlexLeung** [comment]
Seems like this may be related to [#144](https://github.com/apollostack/graphql-server/issues/144)

**Comment by strangegit** [comment]
it works with for me with koa-bodyparser@next

**Comment by DxCx** [comment]
@AlexLeung can you confirm?

**Comment by AlexLeung** [comment]
I cannot confirm.
I ran `npm i --save koa-bodyparser@next` and replaced `import koaBody from 'koa-body';` with `import koaBody from 'koa-bodyparser'`, and am still using Koa 2. Same errors.

**Comment by AlexLeung** [comment]
Here's my code: https://github.com/AlexLeung/ApolloServerKoa_Test/blob/master/server.js

**Comment by AlexLeung** [comment]
Once we figure this out I'd like to make a PR which updates README to tell people **exactly** what packages and imports they should be using.

**Comment by helfer** [comment]
@HriBB can you take a quick look and see if there's any problem with our package here?

**Comment by sweetcharity** [comment]
you have version 5.x of koa-router installed, but you need 7.x. do ```npm install koa-router@next```

**Comment by helfer** [comment]
@AlexLeung can you check if @sweetcharity 's suggestion fixes your issue?

**Comment by AlexLeung** [comment]
It worked. #242

**Comment by AlexLeung** [comment]
https://github.com/apollostack/graphql-server/pull/242/commits/78cefe060c0c0368454c83b80b21191d23c602c0

**Comment by eudaimos** [comment]
@AlexLeung I wanted to resurrect this for those who come across it - Koa has released v2 as `2.1` and is no longer using the `@2` and `@next` tags (although those still work as of today and point to the correct `master` versions now) for _most_ of the packages **except** `koa-router` for which to make this work will need to use `@next` in order to get version `7`+
_I had to learn this one the hard way_
